docs/buildah-build.1.md

```diff
@@ -96,7 +96,37 @@ The value of `[name]` is matched with the following priority order:
 
 **--cache-from**
 
-Images to utilise as potential cache sources. Buildah does not currently support --cache-from so this is a NOOP.
+Repository to utilize as a potential cache source. When specified, Buildah will try to look for
+cache images in the specified repository and will attempt to pull cache images instead of actually
+executing and computing STEPS. Buildah will only attempt to pull previously cached images if they
+are considered as valid cache hits otherwise not.
```

Comment:
```suggestion
executing and computing STEPS. Buildah will only attempt to pull previously cached images if they
are considered as valid cache hits.
```
or just drop the sentence completely, I’m not sure what users are supposed to do with that information.